lib/cisco_node_utils/bgp.rb

```diff
@@ -685,8 +690,11 @@ def router_id
     def router_id=(id)
       # In order to remove a bgp router-id you cannot simply issue
       # 'no bgp router-id'. Dummy-id specified to work around this.
-      dummy_id = '1.2.3.4'
+      # not working on N7K need a valid router_id
```

Comment:
Second this.  This is what I had suggested to you before as well.

https://github.com/cisco/cisco-network-node-utils/pull/195#discussion-diff-51036558